anitya/templates/project_new.html

```diff
@@ -118,8 +118,29 @@ <h1>{{ context }} project</h1>
       {%- endautoescape -%}
     {%- endfor %}
 
+  function add_tooltips(){
+    $("#name").prop('title', "Project upstream name");
+    $("#homepage").prop('title', "Project homepage");
+    $("#version_url").prop('title', "URL where to look for new releases");
+    $("#version_prefix").prop('title', "Prefix filled here will be removed " +
+        "from retrieved version. For example version prefix " +
+        "'v' will convert 'v1.2.3' to '1.2.3'.");
+    $("#regex").prop('title', "Python regex to use for version retrieval. " + 
+        "Matched groups will be concatenated by '.'. " + 
+        "For example regex '(\d)_(\d)_(\d)' applied to '1_2_3' " +
+        "will convert version to '1.2.3'.");
+    $("#insecure").prop('title', "This should be set only when Version URL is HTTP");
```

Comment:
This should be "This should only be set when the version URL is HTTPS and the host does not have a trusted certificate"